engine/runtime/src/test/scala/org/enso/interpreter/test/semantic/PolyglotTest.scala

```diff
@@ -119,7 +122,7 @@ class PolyglotTest extends InterpreterTest {
           |""".stripMargin
       eval(code)
       val count :: Nil = consumeOut
-      count shouldEqual "OK"
+      count shouldEqual "FAIL"
     }
```

Comment:
Shouldn't we instead change the above match to `Polyglot.Polyglot` to preserve the semantics of this tests?

Reply:
That is being tested later so I don't think so.

Reply:
Well, then maybe we should just remove this test? After this change it seems a bit useless IMO

Reply:
Okay, fair